### DISCUSSION TITLE
Nathan Wienert - One

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#352] [Announcements]
- https://onestack.dev/
- https://x.com/one__js
- https://x.com/natebirdman

## Questions

- [x] What is your relationship with uniswap? Have they been beta testers in your tech or something more?
- [x] How did the one stack come about? Are there any apps in the wild that use it?
- [x] What is the one stack?
- [x] Was it tough getting the react native ecosystem working with vite?
- [x] What is zero.dev? How does it fit into one? What is the goal with this type of setup?
- [x] What are some unique things about zero that make building apps with one nicer?
- [x] ...what about RSC?
- [x] The file based device layout is SWEET! What other features like that does one provide to make creating a universal app easier?
- [ ] What is the vite plugin doing?
- [x] You have fully typed routing. How did you accomplish it?
- [x] What are some unique routing features that one has?

### Future

- [ ] What's left to solve to make universal apps easier to build?

**Comment by just-be-dev** [comment]
render modes as part of the filename is nead. `route+ssg.tsx` for example

**Comment by just-be-dev** [comment]
Tamagui was ep 64

**Comment by MattCrBoom** [comment]
rr!
